source/extensions/common/wasm/wasm.cc

```diff
@@ -94,11 +152,39 @@ class SharedData {
   }
 
 private:
+  uint32_t nextQueueToken() {
+    while (true) {
+      uint32_t token = next_queue_token++;
+      if (!token) {
```

Comment:
Nit: since this is a number, `token == 0`.

Reply:
Done